### PR TITLE
Switch the overcloud to use the admin network

### PR DESCRIPTION
**Pull request by jovial**
We are waiting on https://review.openstack.org/#/c/572370/ to land

### CODE REVIEW

File: etc/kayobe/inventory/group_vars/compute/network-interfaces [hunk 1]
```diff
@@ -19,13 +19,11 @@ internal_interface: "{{ oc_provision_interface }}.{{ internal_vlan }}"
 storage_interface: "{{ oc_provision_interface }}.{{ storage_vlan }}"
```
Comment (markgoddard) [inline etc/kayobe/inventory/group_vars/compute/network-interfaces]: Can also remove os_provision_* from here and controllers and storage network-interfaces files.

**Review by markgoddard — CHANGES_REQUESTED** [review]
We're going to need to address this before removing the oc_provision network from hosts:

etc/kayobe/docker.yml-28-# This a private docker registry running on the seed, accessed via overcloud
etc/kayobe/docker.yml-29-# provisioning network.
etc/kayobe/docker.yml:30:# TODO: Use a more sensible network, e.g. devops.
etc/kayobe/docker.yml-31-docker_registry: "10.61.101.6:5000"
etc/kayobe/docker.yml-32-

**Comment by markgoddard** [comment]
We could also remove the IP allocations from network-allocations.yml

**Comment by jovial** [comment]
I've retagged the images with:
```
docker image ls | awk '{if($1 ~ /^10.61.101.6:5000/){test=$1;gsub("10.61.101.6:5000","10.64.1.0:5000",test);print $1 ":" $2, test ":" $2}}' | while read line ; do docker tag $line ; done
```
Some of the intermediate layers have the `<none>` tag, the above one liner created the equivalent with a latest tag (not sure if you can tag something with `<none>`). E.g:

```
10.64.1.0:5000/eod/centos-binary-base                            latest              03f9abe52515        2 months ago        403MB
10.61.101.6:5000/eod/centos-binary-base                          <none>              03f9abe52515        2 months ago        403MB
```

Not sure if this is going to be an issue.

**Comment by markgoddard** [comment]
The retagged images will need to be pushed to the registry so that they can be pulled down by the nodes. Normally we do this from the seed.

**Review by markgoddard — APPROVED** [review]

**Comment by jovial** [comment]
Just realized that I set the docker tags to have the IP of the controller. The seed doesn't yet have an IP allocated for the devops network. Can we manually add an ip allocation to `network_allocations.yml`?

**Comment by markgoddard** [comment]
Yes you can add the node's devops IP, 10.64.101.14.

**Review by markgoddard — APPROVED** [review]

**Comment by markgoddard** [comment]
The docker systemd config is configured by kolla-ansible bootstrap-servers command, in the baremetal role. This is run as part of `kayobe seed|overcloud host configure`.

**Comment by jovial** [comment]
defunct project, closing.

**Pull request closed without merging**